app/src/main/java/com/example/diceroller/ViewState.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.diceroller
+
+sealed interface ViewState {
```

Comment:
I made this. But it still needs a better name. Can you thing of one? Maybe one doesn't exist because 2 sets of concerns are being expressed.

This uses two things called a sealed interface and a sealed class. These things are powerful because they are deterministic. When using `when` on something like this, you get an error if you don't provide instructions for every single case.